src/course/syllabus/pre-app-7/schedule.md

```diff
@@ -0,0 +1,25 @@
+In this Wednesday's workshop we looked at [functions, callbacks and asynchronous JavaScript](https://learn.foundersandcoders.com/workshops/functions-callbacks-async/).
+
+## HTML Forms
+
+_60 minutes_
+
+Learn more about what happens when we submit HTML forms and how to collect their data by complete [this workshop](https://learn.foundersandcoders.com/workshops/html-forms/).
```

Comment:
This workshop doesn't really fit with the other content this week imo. I'm not sure what the best place would be for it though. Conceptually forms are for sending HTTP requests, so in that sense its fits better with the `fetch` stuff